test/utils.jl

```diff
@@ -1,4 +1,26 @@
 @testset "Utils" begin
-  # TODO: readcoda
-  # TODO: compose
+  @testset "readcoda" begin
+    df = readcoda(joinpath(datadir,"juraset.csv"); codanames=(:Cd, :Cu, :Pb, :Co, :Cr, :Ni, :Zn));
+    @test names(df) == ["X", "Y", "Rock", "Land", "Cd|Cu|Pb|Co|Cr|Ni|Zn"]
+    @test size(df) == (359, 5)
+    @test df[1, "Cd|Cu|Pb|Co|Cr|Ni|Zn"] == Composition(Cd=1.74, Cu=25.72, Pb=77.36, Co=9.32, Cr=38.32, Ni=21.32, Zn=92.56)
+
+    df = readcoda(joinpath(datadir,"juraset.csv"); codanames=("Cd", "Cu", "Pb", "Co", "Cr", "Ni", "Zn"));
+    @test names(df) == ["X", "Y", "Rock", "Land", "Cd|Cu|Pb|Co|Cr|Ni|Zn"]
```

Comment:
I wonder if we should use the notation `Cd:CuPb:...:Zn` as opposed to vertical bars. We could change that in a separate PR if necessary. The `:` seems to convey the message of a "ratio" better than `|`.

Reply:
I actually am neither very fond of `|` or of `:`. 
I think both options may lead to usability dificulties like need to type long and difficult to remember names, specially for compositions with a lot of components. Using `|` or `:` would hinder tab-completion as well.
Do you think an additional argument `;comp_name="coda"` on the `readcoda` and `compose` functions could be a good way to set the column name? (thought maybe "coda" could be a good default name)

Reply:
Love the idea. :100: We just need a better name for the option. We used `codanames` to specify the individual components, and so `comp_name` would be somewhat confusing. What do you think of making `codanames` a pair like `codanames=("Cd","Cu","Pb") => "coda"`? If the user passes a tuple, we create the pair with the default `"coda"` value as you suggested.

Reply:
I think using a `codanames=("Cd","Cu","Pb") => "coda"` would be a simple, great solution! As soon as #12 is merged, I can try to make the modifications, if you think it is a good idea.

Reply:
Awesome! I've re-triggered the build there.